roles/build-release/tasks/setup.yaml

```diff
@@ -1,15 +1,4 @@
 ---
-- name: Install antsibull from current source in a virtualenv with poetry
-  command: poetry install
-  args:
-    chdir: "{{ playbook_dir | dirname }}"
-
-# Existing poetry installs are not updated and it could lead to stale deps
-- name: Ensure poetry virtualenv is up to date
-  command: poetry update
-  args:
-    chdir: "{{ playbook_dir | dirname }}"
```

Comment:
I see where you are going but I'm a bit torn on this -- on one hand, antsibull is assumed to be installed but the role was previously taking care of that.

Should the role create a virtualenv in which it'd install antsibull (from source or from pypi) instead ?